test/unit/utils-source-parsing-specs.js

```diff
@@ -470,5 +470,111 @@ describe('utils/source-parsing.js', function () {
         path: '',
       });
     });
+
+    it('should convert xml to json for Newline', function () {
```

Comment:
Thanks for adding a new test as well! I think that since this test is specifically for the newline conversion functionality, you can probably remove all other attributes that don't have the newline symbols, which would keep the test shorter.

Reply:
removed. Thanks, @eglitise for the advice.